### PR TITLE
Bump requested version of net-exporter

### PR DESCRIPTION
**Pull request by mcharriere**
Signed-off-by: Matias Charriere <matias@giantswarm.io>

### Checklist
- [ ] Roadmap issue created
- [ ] Release uses latest stable Flatcar
- [ ] Release uses latest Kubernetes patch version

### CODE REVIEW

**Comment by mcharriere** [comment]
towards https://github.com/giantswarm/adidas/issues/1142

File: azure/requests.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ releases:
   - name: coredns
     version: ">= 1.14.2"
   - name: net-exporter
-    version: ">= 1.16.1"
+    version: ">= 1.16.2"
```
Comment (whites11) [inline azure/requests.yaml:7]: unfortunately there is already an azure release > 19.0.1.
Please change the `--name` value at line 2 with something like:

```
- name "> 19.0.1 <19.999.999"
```

Comment (mcharriere) [reply azure/requests.yaml:7]: thanks. it's also complaining about v20-beta1. I'll try fixing it after lunch

Comment (whites11) [reply azure/requests.yaml:7]: that is what I was talking about.

File: azure/requests.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 releases:
-- name: "> 19.0.2"
+- name: "> 20.0.0"
```
[the remark this answers is not in the document]
Comment (mcharriere) [reply azure/requests.yaml:2]: nope, 19.0.2 also complained about v20-beta

**Review by whites11 — APPROVED** [review]

**Pull request merged**